Sources/OpenAPI/Generated/Entities/MetricsInsight.swift

```diff
@@ -8,7 +8,7 @@ public struct MetricsInsight: Codable {
 	public var latestVersion: String?
 	public var metric: String?
 	public var summaryString: String?
-	public var referenceVersions: [String]?
+	public var referenceVersions: String?
```

Comment:
Be aware of the custom change in https://github.com/AvdLee/appstoreconnect-swift-sdk/pull/256/files.